README.md

```diff
@@ -1,2 +1,26 @@
 # discourse-tests
 Automated tests for discourse.mozilla-community.org
+
+How to set up and run Discourse tests locally
+
+This repository contains Selenium tests used to test:
+* staging: https://discourse.staging.paas.mozilla.community/
+* production: https://discourse.mozilla-community.org/
+
+###You will need to install the following:
+* Git
+If you have cloned this project already then you can skip this! GitHub has excellent guides for Windows, Mac OS X, andLinux.
+
+* Python
+Before you will be able to run these tests you will need to have Python 2.6.8+ installed.
```

Comment:
Markdown doesn't treat this as a new line, please make sure that the rendered doc looks OK.